src/Gui/InventorNavigationStyle.cpp

```diff
@@ -113,13 +113,13 @@ SbBool InventorNavigationStyle::processSoEvent(const SoEvent * const ev)
 
     // Keyboard handling
     if (type.isDerivedFrom(SoKeyboardEvent::getClassTypeId())) {
-        const SoKeyboardEvent * const event = static_cast<const SoKeyboardEvent *>(ev);
+        auto event = static_cast<const SoKeyboardEvent *>(ev);
         processed = processKeyboardEvent(event);
     }
 
     // Mouse Button / Spaceball Button handling
     if (type.isDerivedFrom(SoMouseButtonEvent::getClassTypeId())) {
-        const SoMouseButtonEvent * const event = (const SoMouseButtonEvent *) ev;
+        const auto event = (const SoMouseButtonEvent *) ev;
```

Comment:
(Side note for future work: we should get rid of all these old-style casts, too [in a separate PR!])

Reply:
I'm spending some time now working through these existing PRs, I don't know if I'll get to that one today, but I do plan on getting to it.

Reply:
I'll readapt it again and ping you when it's finished